api/views.py

```diff
@@ -756,13 +757,13 @@ class StructureSequenceParser(views.APIView):
     curl -X POST -F "pdb_file=@myfile.pdb" http://gpcrdb.org/services/structure/parse_pdb
     """
     parser_classes = (FileUploadParser,)
-    renderer_classes =(JSONRenderer)
+    renderer_classes = (JSONRenderer, )
 
     def post(self, request):
-
-        root, ext = os.path.splitext(request.FILES['pdb_file'].name)
-        header = parse_pdb_header(request.FILES['pdb_file'])
-        parser = SequenceParser(request.FILES['pdb_file'])
+        root, ext = os.path.splitext(request._request.FILES['pdb_file'].name)
```

Comment:
# root, ext = os.path.splitext(request._request.FILES['pdb_file'].name)

Reply:
I think this can be commented out?

Reply:
Indeed, just legacy code that I also used for debugging. I've commented them.